### PR TITLE
refactor commits

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/main/java/com/foundation/trello/util/Regex.java [hunk 1]
```diff
@@ -10,45 +11,39 @@
...
+            ourInstance = new Regex();
         }
         return ourInstance;
     }
 
     /**
      * This method returns a new endpoint.
      *
+     * @param endpoint The string parameter defines the endpoint input.
+     * @param id       The id parameter defines the list of id.
```
Comment (MadayAlcala) [inline src/main/java/com/foundation/trello/util/Regex.java]: Please delete tab unnecessary tab.

File: src/main/java/com/foundation/trello/util/Regex.java [hunk 2]
```diff
@@ -10,45 +11,39 @@
...
+        for (Map.Entry<String, String> entry : id.entrySet()) {
+            Pattern pattern = Pattern.compile("\\{" + entry.getKey() + "\\}");
+            Matcher matcher = pattern.matcher(endpoint);
+            endpoint = matcher.replaceAll(entry.getValue());
         }
-        return string;
+        return endpoint;
     }
 }
+
```
Comment (LimbertVargas) [inline src/main/java/com/foundation/trello/util/Regex.java]: please delete this line

File: src/main/java/com/foundation/trello/util/Regex.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/foundation/trello/util/Regex.java]: I fixed it.

File: src/main/java/com/foundation/trello/util/Regex.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/foundation/trello/util/Regex.java]: I fixed it.

File: src/test/java/com/foundation/trello/hook/ChecklistHook.java [hunk 3]
```diff
@@ -60,7 +62,7 @@ public void beforeScenario() {
         requestManager.setData(data);
         response = requestManager.makeRequest();
         Log.getInstance().getLog().info(response);
-        context.setId(response.jsonPath().get("id"));
+        context.getMapIds().put("idChecklist", response.jsonPath().get("id"));
     }
 }
```
Comment (LimbertVargas) [inline src/test/java/com/foundation/trello/hook/ChecklistHook.java]: please delete this line

Comment (JosueRodriguezGarcia) [reply src/test/java/com/foundation/trello/hook/ChecklistHook.java]: I fixed it.

**Review by MadayAlcala — APPROVED** [review]
Great job!

**Review by LimbertVargas — APPROVED** [review]

**Review by raulchoque-eng — APPROVED** [review]

**Pull request merged**